src/patternfly/components/VerticalNav/vertical-nav-link-icon.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="pf-c-vertical-nav__link__icon {{ pf-c-vertical-nav--modifier }}">
```

Comment:
In this case, since icons are required and potentially not coming from fontawesome, we should add `aria-hidden="true"` to the `<span>` so that this element is ignored by screen readers.